Pokemon/Pokemon/PokemonList.m

```diff
@@ -0,0 +1,39 @@
+//
+//  PokemonList.m
+//  Pokemon
+//
+//  Created by Brian Blanco on 7/31/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "PokemonList.h"
+#import "PokemonLibrary.h"
+
+@implementation PokemonList
+
+- (instancetype)initWithIndex:(NSUInteger)index {
+    self = [super init];
+    if (self) {
+        PokemonLibrary *pokemonLibrary = [[PokemonLibrary alloc] init];
```

Comment:
This class shouldn't have to know about `PokemonLibrary`. Instead of passing a list index, you could just pass the `pokemonListDictionary` you're pulling out of the library below (on line 20).